### PR TITLE
systemd-dissect: add page

### PR DESCRIPTION
**Pull request by quantumflo**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/systemd-dissect.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# systemd-dissect
+
+
```
Comment (gutjuri) [inline pages/linux/systemd-dissect.md:3]:
```suggestion

```

One empty line less :P

File: pages/linux/systemd-dissect.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+> Tool for introspecting and interacting with file system OS disk images, specifically Discoverable Disk Images (DDIs).
+> More information: <https://www.freedesktop.org/software/systemd/man/latest/systemd-dissect.html>.
+
+- Show General Image Information about the OS image:
```
Comment (gutjuri) [inline pages/linux/systemd-dissect.md:7]:
```suggestion
- Show general image information about the OS image:
```

File: pages/linux/systemd-dissect.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+`systemd-dissect {{myimage.raw}}`
+
+- Mount an OS Image:
```
Comment (gutjuri) [inline pages/linux/systemd-dissect.md:11]:
```suggestion
- Mount an OS image:
```

File: pages/linux/systemd-dissect.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# systemd-dissect
+
+
+> Tool for introspecting and interacting with file system OS disk images, specifically Discoverable Disk Images (DDIs).
+> More information: <https://www.freedesktop.org/software/systemd/man/latest/systemd-dissect.html>.
+
+- Show General Image Information about the OS image:
+
+`systemd-dissect {{myimage.raw}}`
```
Comment (gutjuri) [inline pages/linux/systemd-dissect.md:9]:
```suggestion
`systemd-dissect {{path/to/image.raw}}`
```
Assuming a relative path is allowed here.

File: pages/linux/systemd-dissect.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+- Mount an OS Image:
+
+`systemd-dissect --mount {{myimage.raw}} {{/mnt/myimage}}`
```
Comment (gutjuri) [inline pages/linux/systemd-dissect.md:13]:
```suggestion
`systemd-dissect --mount {{path/to/image.raw}} {{/mnt/image}}`
```

File: pages/linux/systemd-dissect.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+- Unmount an OS Image:
+
+`systemd-dissect --umount {{/mnt/myimage}}`
```
Comment (gutjuri) [inline pages/linux/systemd-dissect.md:17]:
```suggestion
- Unmount an OS image:

`systemd-dissect --umount {{/mnt/image}}`
```

File: pages/linux/systemd-dissect.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+- List Files in an Image:
+
+`systemd-dissect --list {{myimage.raw}}`
```
Comment (gutjuri) [inline pages/linux/systemd-dissect.md:21]:
```suggestion
- List Files in an image:

`systemd-dissect --list {{path/to/image.raw}}`
```

File: pages/linux/systemd-dissect.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+- Attach an OS Image:
+
+`systemd-dissect --attach {{myimage.raw}}`
```
Comment (gutjuri) [inline pages/linux/systemd-dissect.md:25]:
```suggestion
- Attach an OS image:

`systemd-dissect --attach {{path/to/image.raw}}`
```

**Review by gutjuri — COMMENTED** [review]
Thanks for your contribution. I've added some suggestions.

File: pages/linux/systemd-dissect.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+`systemd-dissect --mount {{path/to/image.raw}} {{/mnt/image}}`
+
+- Unmount an OS image:
+
+`systemd-dissect --umount {{/mnt/image}}`
+
+- List files in an image:
+
+`systemd-dissect --list {{myimage.raw}}`
```
Comment (gutjuri) [inline pages/linux/systemd-dissect.md:20]:
```suggestion
`systemd-dissect --list {{path/to/image.raw}}`
```

File: pages/linux/systemd-dissect.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
+# systemd-dissect
+
+> Tool for introspecting and interacting with file system OS disk images, specifically Discoverable Disk Images (DDIs).
```
Comment (acuteenvy) [inline pages/linux/systemd-dissect.md:3]:
```suggestion
> Introspect and interact with file system OS disk images, specifically Discoverable Disk Images (DDIs).
```

File: pages/linux/systemd-dissect.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+
+`systemd-dissect --umount {{/mnt/image}}`
+
+- List files in an image:
+
+`systemd-dissect --list {{myimage.raw}}`
+
+- Attach an OS image:
+
+`systemd-dissect --attach {{path/to/image.raw}}`
```
Comment (acuteenvy) [inline pages/linux/systemd-dissect.md:24]:
```suggestion
- Attach an OS image to an automatically allocated loopback block device and print its path:

`systemd-dissect --attach {{path/to/image.raw}}`

- Detach an OS image from a loopback block device:

`systemd-dissect --detach {{path/to/image.raw}}`
```

**Review by gutjuri — APPROVED** [review]

**Pull request merged**